### PR TITLE
Redigieren

### LINKED ISSUE #441: Make sure Leas changes were kept
In PR #192 some changes by @LeaKruehler seem to have been reverted. For example, 

https://github.com/hbrs-cse/Modellbildung-und-Simulation/blob/a58cabb2e0b0f226122fcc10c08f6754d39aa09e/content/00_einleitung/01_motivation.ipynb?short_path=0b12a8d#L38

Go through #192 and revert, what's necessary.

### PR DESCRIPTION
**Pull request by LeaKruehler**
Korrekturen Rechtschreibung
closes #441 

# Description

## Checklist

 - [ ] This pull request is associated to an issue
 - [ ] All used resources use a CC BY-SA 4.0 license or less and are marked appropriately, see our [contribution guide](https://github.com/hbrs-cse/Modellbildung-und-Simulation/blob/master/CONTRIBUTING.md).
 - [ ] All used resources have been added to [list_of_all_resources.md](https://github.com/hbrs-cse/Modellbildung-und-Simulation/blob/master/docs/List_of_all_resources.md)
 - [ ] New exercises include learning requirements and learning goals.
 - [ ] Add someone else as reviewer and wait for approval before merging.

### CODE REVIEW

**Comment by LeaKruehler** [comment]
@PhiSpel Can you help me here? I have no clue, why the CircleCI test are failing. 
The details say, that the problem is in line 33 of "erste Schritte in Matlab" but i didn't change anything in that line. Maybe its something about the link?

**Comment by joergbrech** [comment]
The link https://hackernoon.com/10-reasons-to-learn-python-in-2018 appears to be dead. https://app.circleci.com/pipelines/github/hbrs-cse/Modellbildung-und-Simulation/1581/workflows/35f8baad-0532-4d3f-a095-89b120d743b9/jobs/3027?invite=true#step-104-175

We have to remove it or replace it with another python reference.

File: content/10_dft/03_der_doppler_effekt.ipynb [hunk 1]
```diff
@@ -28,7 +28,7 @@
     "Der Inhalt dieser Seite lässt sich nicht interaktiv auf dieser Seite ausführen. \n",
     "```\n",
     "\n",
-    "In [dem folgenden Video](https://www.youtube.com/watch?v=a3RfULw7aAY) ist kurz ein Verkehrsschild zu sehen: Hier ist eine Maximalgeschwindigkeit von 50 mph, bzw. ca. 80 km/h vorgeschrieben.\n",
+    "In [dem folgenden Video](https://www.youtube.com/watch?v=a3RfULw7aAY) ist kurz ein Verkehrsschild zu sehen: Hier ist eine Maximalgeschwindigkeit von 50 mp/h, bzw. ca. 80 km/h vorgeschrieben.\n",
```
Comment (PhiSpel) [inline content/10_dft/03_der_doppler_effekt.ipynb:31]: mph is correct

**Comment by PhiSpel** [comment]
@LeaKruehler if you agree, you can merge #440, with which this PR should work.

File: content/10_dft/03_der_doppler_effekt.ipynb (hunk 1 above)
Comment (LeaKruehler) [reply content/10_dft/03_der_doppler_effekt.ipynb]: imperial measurements are just weird... i changed it back :) thanks!

**Comment by LeaKruehler** [comment]
Circle CI is still failing. This times its because of the links from the slacklineverband which seem to be unsafe? The links are used as a reference in Aufgabe 4. Do we need new links or shall we just remove them?

**Comment by PhiSpel** [comment]
I took the liberty of removing it, it is not necessary for the understanding. A pity that slacklineverband.com does not follow the https protocoll atm, those things are annoying..

**Review by PhiSpel — DISMISSED** [review]

**Pull request merged**